### PR TITLE
fix: update add room feature

### PR DESCRIPTION
**Pull request by ma-za-kpe**
For instance, 
(DONE)you have a lazy row in which there are single items. Why don't you use a column
with a vertical scroll modifier? 

(DONE)And I think you can achieve the screen without a constraint layout (I may be wrong BTW).
Margin and padding can be added using our local spacing variable, as most of them exist.

(DONE)The icon button is not working (on click), 

(DONE)and you display a list of 100+ elements that are the same image obtained from local 
resources. 

(DONE)the text fields are of type password, and you use hard-coded values in the ViewModel 
for strings, etc.

tw

add

### CODE REVIEW

File: ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt [hunk 1]
```diff
@@ -80,6 +85,13 @@ fun AddRoomScreen(
     }
     val focusManager: FocusManager = LocalFocusManager.current
 
+    var selectImages by remember { mutableStateOf(listOf<Uri>()) }
+    val galleryLauncher =
+        rememberLauncherForActivityResult(ActivityResultContracts.GetMultipleContents()) {
+            selectImages = it
```
Comment (kamdaou-amalitech) [inline ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt:91]: Can you create a method in the ViewModel responsible for adding the selected image in your uiState? then in this screen, you'll call it using something like `viewModel.addImage(it)`. Remember that it's a list; right now you're replacing the content of the list with selected items, so you'll lose all previous images every time you select new ones. Can you instead add them to previously selected images and add a small delete icon button on every image to allow users to remove an image from the list if they wish?

File: ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt [hunk 2]
```diff
@@ -421,7 +430,7 @@ fun RoomTextFieldPreview() {
             mutableStateOf("")
         }
         RoomTextField(
-            placeholder = "password",
+            placeholder = "Room Name",
```
Comment (kamdaou-amalitech) [inline ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt:433]: For accessibility (i18n) purposes, we should use a string resource here as well as this text cannot be translated if we decide to translate the app into another language. Please check also on line 155, 311, 336 and add contentDescription. The app may be used by a blind person and they will use voice commands to navigate in the app, so they will need a content description.

Comment (ma-za-kpe) [reply ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt]: Done, PTAL

File: ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt (hunk 1 above)
Comment (ma-za-kpe) [reply ui/admin/src/main/java/com/amalitech/admin/room/AddRoomScreen.kt]: Done, PTAL.

**Review by kamdaou-amalitech — APPROVED** [review]

**Pull request merged**